packages/core/src/Source.ts

```diff
@@ -230,7 +235,12 @@ export default class Source {
   }
 
   async restart() {
+    console.log(this.#pluginErrors);
```

Comment:
Do you want this? If so should it be console.error instead?

Reply:
no...will remove